utils/config/daos_server.yml

```diff
@@ -185,9 +185,9 @@
 ##
 ## Hugepages are mandatory with NVME SSD configured and optional without so
 ## that when NVMe isn't configured, daos will allocate normal memory pages
-## (not hugepages) if nr_hugepages is 0.
+## (and disable use of hugepages) if nr_hugepages is -1.
```

Comment:
This change seems to imply that the default is now `-1`, but the change to `DefaultServer()` seems to contradict this.

Reply:
I don't see how this implies that the default is -1 but I will try to make the wording more obvious.

Reply:
done